### PR TITLE
Fix Permission screen and Recovery screen UI

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
On short screens (where height is < 480px) the CTA is not visible and the user has to scroll to see it.
Screen Shot 2019-04-24 at 11 50 24

Screen Shot 2019-04-24 at 11 50 43

# Solution
Screen Shot 2019-04-24 at 11 49 59
In the screen above, I made the graphic div responsive so that it becomes smaller if the screen is smaller.
If the screen is shorter than 480px, the div shrinks a bit more, the circle becomes smaller and the margin of the text is also reduced

Screen Shot 2019-04-24 at 11 49 24
In the screen above, for screens shorter than 480px, the text becomes extra small.

# Bonus
Removed collapsible tips
Screen Shot 2019-04-24 at 11 50 30
Screen Shot 2019-04-24 at 11 49 59

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://628-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CameraPermissions/Primer/style.css [hunk 1]
```diff
@@ -35,15 +57,10 @@
...
   position: relative;
-  width: 270*@unit-small;
-
-  @media (--small-viewport) {
-    height: 100*@unit-small;
-    width: 224*@unit-small;
-  }
+  flex: 0 1 100%;
+  /* aspect ratio of responsive div */
+  padding-top: 44%;
```
Comment (rfreitas) [inline src/components/CameraPermissions/Primer/style.css]: where does the 44% come from? some calculation or eyeballing?

Comment (stefaniacardenas) [reply src/components/CameraPermissions/Primer/style.css]: The original height value divided by the original width value. 121/270 = 0.44

Comment (rfreitas) [reply src/components/CameraPermissions/Primer/style.css]: can you put that in the comment? it gives some clues of where it comes from

Comment (stefaniacardenas) [reply src/components/CameraPermissions/Primer/style.css]: Good point. Will add 👍

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-628-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://628-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CameraPermissions/Primer/index.js [hunk 2]
```diff
@@ -9,19 +8,15 @@ import style from './style.css'
 import { localised } from '../../../locales'
 
 const Permissions = ({onNext, translate}) => (
-  <div className={style.container}>
-    <Title title={translate('webcam_permissions.allow_access')} />
-    <div className={theme.thickWrapper}>
-      {translate('webcam_permissions.enable_webcam_for_selfie')}
+  <div className={`${style.container} ${theme.fullHeightContainer}`}>
```
Comment (josokinas) [inline src/components/CameraPermissions/Primer/index.js]: I see the app is using classnames, would you want to use it here instead?

**Pull request merged**